pkg/infrastructure/services/resources.go

```diff
@@ -197,10 +200,13 @@ func (s *serviceDeployer) applyKafkaConfigurations(deployment *appsv1.Deployment
 	if len(URI) > 0 {
 		framework.SetEnvVar(enableEventsEnvKey, "true", &deployment.Spec.Template.Spec.Containers[0])
 		for _, kafkaTopic := range s.definition.KafkaTopics {
-			framework.SetEnvVar(fromKafkaTopicToQuarkusEnvVar(kafkaTopic), URI, &deployment.Spec.Template.Spec.Containers[0])
+			appProps[fromKafkaTopicToQuarkusAppProp(kafkaTopic)] = URI
 		}
-		for _, kafkaEnv := range quarkusBootstrapEnvVars {
-			framework.SetEnvVar(kafkaEnv, URI, &deployment.Spec.Template.Spec.Containers[0])
+		framework.SetEnvVar(quarkusBootstrapEnvVar, URI, &deployment.Spec.Template.Spec.Containers[0])
```

Comment:
Why setting the env var if we set the app prop just after ?

Reply:
It seems the env is still needed by [job service](https://github.com/kiegroup/kogito-images/blob/7a13116a98b7f6de9fba2ff61975106187ddc2c1/modules/kogito-jobs-service/added/launch/kogito-jobs-service.sh#L38).

Reply:
ok

This one is for Quarkus right ?
We should also not set it for Spring Boot or ?

Reply:
Corrected. Thanks.